pkgs/development/python-modules/flammkuchen/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, pkgs, buildPythonPackage, fetchPypi, isPy3k
+, numpy
+, scipy
+, tables
+, pandas
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "flammkuchen";
+  version = "0.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "e9aab9b229ace70d879b85618a9ce0e88dd6ce35d4dbcdfd60c6b61c33a1b4fb";
+  };
+
+  checkInputs = [
+    nose
+  ];
+
+  propagatedBuildInputs = [
+    numpy
+    scipy
+    tables
+    pandas
+  ];
```

Comment:
```suggestion
  ] ++ lib.optionals isPy27 [ configparser ];
```

otherwise you get:
```
warning: ignoring the user-specified setting 'sandbox', because it is a restricted setting and you are not a trusted user
builder for '/nix/store/643f846aw8mdlblbx738am6hyhgayslc-python2.7-flammkuchen-0.9.1.drv' failed with exit code 1; last 10 log lines:
      from configparser import ConfigParser
  ImportError: No module named configparser


  ----------------------------------------------------------------------
  Ran 1 test in 0.000s

  FAILED (errors=1)
  Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
  error: Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
cannot build derivation '/nix/store/fqpy9j40cw5dn0m9sdclaricw7z9aj1n-env.drv': 1 dependencies couldn't be built
[17 built (1 failed), 19 copied (228.5 MiB), 45.0 MiB DL]
error: build of '/nix/store/fqpy9j40cw5dn0m9sdclaricw7z9aj1n-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/72353
1 package failed to build:
python27Packages.flammkuchen
```

Reply:
thanks! committed